mne/io/fiff/tests/test_raw.py

```diff
@@ -888,9 +888,16 @@ def test_hilbert():
     picks = picks_meg[:4]
 
     raw2 = raw.copy()
+    raw3 = raw.copy()
+    raw4 = raw.copy()
     raw.apply_hilbert(picks)
     raw2.apply_hilbert(picks, envelope=True, n_jobs=2)
 
+    # Test custom N
+    raw3.apply_hilbert(picks, N=raw3.n_times + 100)
+    assert_equal(raw3._data.shape, raw._data.shape)
```

Comment:
OK so how do you think this should be incorporated into the tests then? Just allow for a larger difference in the call to `allclose`? And maybe include a line in the docstring that zero padding will change things but not too much for bandpassed signals?